integration-tests/README.md

```diff
@@ -30,6 +33,11 @@ aws --profile default configure set aws_access_key_id "123"
 aws --profile default configure set aws_secret_access_key "456"
 ```
 
+Build the contract:
+```bash
+cargo build -p mpc-contract --target wasm32-unknown-unknown --release --target-dir target/seperate_wasm
```

Comment:
I thought we automated this. cc @ChaoticTempest

Reply:
yup, it's no longer necessary to specify `-p mpc-contract` and `--target-dir ...`

Reply:
Yeah, looks it's fixed in latest develop